code/unit_tests/backgrounds.dm

```diff
@@ -0,0 +1,99 @@
+/datum/unit_test/background
+	name = "BACKGROUND - All Species Background Values Shall Be Of Valid Types And Length"
+
+/datum/unit_test/background/start_test()
+
+	var/list/all_background_tokens = global.using_map.get_background_categories()
+
+	var/fails = 0
+	for(var/species_name in get_all_species())
+		var/decl/species/species = get_species_by_key(species_name)
```

Comment:
kind of wondering if this should be on `/decl/species/validate()` instead?

edit: oh this already existed, fair enough